Main.java

```diff
@@ -84,9 +85,8 @@ public static void GUI() {
     success2 = new JLabel("");
     success2.setBounds(280, 50, 300, 25);
     panel.add(success2);
-    frame.setSize(1080, 720);
 
-    Environment currentEnvironment = getEnvironment();
+    currentEnvironment = getEnvironment();
```

Comment:
In the code below where you're setting the forground colour, etc. That would be a good opportunity to use an Interface.  Something like:
```
// Attributes of something that can be displayed on the screen.
public interface IDisplayable {
  Color foregroundColor();
  Color backgroundColor();
  // ... etc.
}
```

Then `Environment` can implement `IDisplayable` and each evironment can control its own colour scheme; instead of this main function having knowledge of specific environments.